vclib-openid/src/commonMain/kotlin/at/asitplus/wallet/lib/oidc/helpers/ClientIdSchemeParameters.kt

```diff
@@ -0,0 +1,33 @@
+package at.asitplus.wallet.lib.oidc.helpers
+
+import at.asitplus.crypto.datatypes.pki.X509Certificate
+import at.asitplus.wallet.lib.oidc.OpenIdConstants
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed interface ClientIdSchemeParameters {
```

Comment:
See https://github.com/a-sit-plus/kmm-vc-library/pull/67#discussion_r1610144181

Reply:
Using the backing field approach from the other discussion is not a solution here:

The following implementation results in an error on `OtherClientIdSchemeParameters`: "This class is not serializable automatically because it has primary constructor parameters that are not properties"
```
@Serializable
sealed class ClientIdSchemeParameters(
    open val clientIdScheme: OpenIdConstants.ClientIdScheme
) {

    @Serializable
    class OtherClientIdSchemeParameters(clientIdScheme: OpenIdConstants.ClientIdScheme) :
        ClientIdSchemeParameters(
            clientIdScheme = clientIdScheme,
        )
}
```

Reply:
resolvable?